lib/get-config.js

```diff
@@ -51,20 +44,11 @@ export async function resolveProjectConfig(workingDir, options) {
   let configPath;
 
   if (options.configPath) {
-    configPath = path.resolve(workingDir, options.configPath);
-
-    try {
-      // Making sure that the filePath exists, before requiring it directly this is
-      // needed in order to ensure that we only squelch module missing errors for
-      // the config path itself (and not other files that the config path may
-      // require itself)
-      require.resolve(configPath);
-    } catch (error) {
-      let isModuleMissingError = ALLOWED_ERROR_CODES.has(error.code);
-      if (!isModuleMissingError) {
-        throw error;
-      }
+    configPath = findUpSync(options.configPath, {
```

Comment:
This isn't quite right, the intent of the code here is that anyone explicitly passing `--config-path` does **not** get "Find up" behavior (and this change here breaks that intent). The real bug is that `--config-path` is explicitly defaulted in our bin scripts arguments, so we can't fall into the fallback `else` clause down below that already uses `findUp`. 

Chatted with @scalvert about this, and he is going to update and add some additional tests to cover those cases.

Reply:
Updated as per our chat.